### PR TITLE
Fixes NPE after renaming class name from ProjectManager. (fixes #1464)

### LINKED ISSUE #1464: Refactor rename class error
Sometimes after renaming a class I get this error.  I don't notice the actual refactors/renames was failed, I just get this message.

image

I use a quite recent dev build (https://github.com/fdorg/flashdevelop/commit/120f067995):
image

### PR DESCRIPTION
**Pull request by SlavaRa**

### CODE REVIEW

File: External/Plugins/CodeRefactor/Provider/ICommandFactory.cs [hunk 1]
```diff
@@ -122,23 +122,23 @@ public interface ICommandFactory
         /// Outputs found results.
         /// Uses the current text location as the declaration target.
         /// </summary>
-        Command CreateRenameCommand();
+        Command CreateRenameCommandAndExecute();
```
Comment (Neverbirth) [inline External/Plugins/CodeRefactor/Provider/ICommandFactory.cs]: This change feels strange. Why AndExecute? Is the command executed by itself? At least in the Factory itself is not run. I would say it's not right if this command behaves in a way all the other commands in a different one.

Comment (SlavaRa) [inline External/Plugins/CodeRefactor/Provider/ICommandFactory.cs]: Yes, it starts execution right after the creation. This appeared after adding "Inline renaming". I'll fix it, but it takes time

**Comment by Neverbirth** [comment]
Conflicts now.

**Comment by SlavaRa** [comment]
5 minutes please

**Comment by SlavaRa** [comment]
fixed

File: External/Plugins/CodeRefactor/Provider/ICommandFactory.cs (hunk 1 above)
Comment (Neverbirth) [inline External/Plugins/CodeRefactor/Provider/ICommandFactory.cs]: Ok, I will merge it, but I would say it definitely needs to be changed.

Comment (SlavaRa) [inline External/Plugins/CodeRefactor/Provider/ICommandFactory.cs]: I completely agree with you, I will correct in the near future

**Pull request merged**